apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -179,19 +159,25 @@ export default class AnimationPickerBody extends React.Component {
     ));
   }
 
-  animationItemsRendering(animations) {
+  animationItemsRendering(animations, multiSelect) {
     return animations.map(animationProps => (
       <AnimationPickerListItem
         key={animationProps.sourceUrl}
         label={this.props.hideAnimationNames ? undefined : animationProps.name}
         animationProps={animationProps}
-        onClick={this.props.onPickLibraryAnimation.bind(this, animationProps)}
+        onClick={() =>
+          this.props.onPickLibraryAnimation(animationProps, multiSelect)
+        }
         playAnimations={this.props.playAnimations}
+        selected={this.props.selectedAnimations.some(
+          e => e.sourceUrl === animationProps.sourceUrl
+        )}
       />
     ));
   }
 
   render() {
+    const multiSelectEnabled = experiments.isEnabled(experiments.MULTISELECT);
```

Comment:
move checking this experiment to `componentDidMount` to avoid performance issues (calling [experiments.isEnabled](https://github.com/code-dot-org/code-dot-org/blob/ed965b3eaa34570f39ea6f488efdb919b647101a/apps/src/util/experiments.js#L113) checks localStorage, which is expensive to do in a component render)

Reply:
we should also probably then feed this value as a prop into `AnimationPickerListItem` when we render it from this component so it doesn't have to check as well